spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/XsuaaServiceConfigurationDefaultTest.java

```diff
@@ -0,0 +1,41 @@
+package com.sap.cloud.security.xsuaa;
+
+import com.sap.cloud.security.config.cf.CFConstants;
+import org.junit.Before;
```

Comment:
Was there a specific reason why you chose junit4 instead of junit5? If not I would suggest to write all new tests in Junit5.

Reply:
Here I just copied the test cases from XsuaaServicesParserTest, which were written in Junit 4. Furthermore i've understood, that EnvironmentVariables Rule didn't worked with JUnit5?

Reply:
the `Rule` itself doesn't work with junit5 it needs to be migrated to `SystemStubs` `EnvironmentVariables`, which is another library, but  Junit5 `EnvironmentVariables` works overall with Junit5 the problem was the test isolation for different environment variable settings, which requires then `reuseForks` to be set to false in surefire plugin.